docs/reference/api.rst

```diff
@@ -20,6 +20,12 @@ Constants
     :members:
     :undoc-members:
 
+.. data:: inmanta_lsm.const.LSM_ENV_VARS
```

Comment:
I think you can automatically extract the docstring from `inmanta_lsm.const.LSM_ENV_VARS` using the `.. automodule::`. But I am not sure. Same for the type annotation.

Reply:
I tried some automodule and it didn't work, I already tried autodata and it didn't work neither. But shouldn't inmanta_lsm be imported somewhere for this to work?